### DISCUSSION TITLE
how can i connect to my gmail acocunt

### DISCUSSION THREAD

**Discussion by udikatz** [FlowiseAI/Flowise#411] [Q&A]
and ask questions about my emails?

**Comment by kushalhalder** [comment]
This looks like a RAG and QnA implementation. 
Thinking out loud. You will need to do the following:
1. Build a RAG implementation with any Vector embedding storage workflow. 
2. Ingest historical emails into the workflow
3. Ingest new emails in realtime
4. Plug the Vector database with an LLM model to answer your questions.
